src/i18n/i18nConfig.ts

```diff
@@ -21,6 +30,9 @@ i18n
     //debug: true,
     fallbackLng: "en",
     resources,
+    interpolation: {
+      escapeValue: false,
```

Comment:
Looking into this, I think it’s better to delete so that default will be `escapedValue: true`. The docs default to using it because [React escapes HTML by default](https://stackoverflow.com/questions/33644499/what-does-it-mean-when-they-say-react-is-xss-protected). However, [as noted here](https://stackoverflow.com/questions/33644499/what-does-it-mean-when-they-say-react-is-xss-protected#comment92875465_33644981), there are other ways to inject malicious code that React does not protect against. Since we’re using strings in a variety of ways and since it doesn’t change the behavior I don’t see a reason to not escape by default, unless there’s some performance difference or something I’m missing. Thanks for calling this out!